src/index.ts

```diff
@@ -74,57 +138,76 @@ import { IconAddBorder, IconStretch, IconAddBackground, IconPicture } from '@cod
  *                           also can contain any additional data that will be saved and passed back
```

Comment:
docs in typedef is not needed anymore. Make sure you moved all the docs, then remove it